### DISCUSSION TITLE
Linux command string for Linux has no binaries

### DISCUSSION THREAD

**Discussion by LinuxNu-B** [phcode-dev/phoenix#1762] [Q&A]
I copied and pasted:
wget -qO- https://updates.phcode.io/linux/installer.sh | bash
to my terminal.
This is what gets returned: 
Using temporary directory /tmp/tmp.QxumLmcIqy for processing
Fetching the latest release information from phcode-dev/phoenix-desktop...
Current GLIBC version: 2.28
Searching for a compatible binary...
No compatible binary found for the current GLIBC version (2.28). Exiting installation.

I'm using MX Linux
19.4 based on Debian 10.13

**Comment by charlypa** [comment]
Hi @LinuxNu-B,

It looks like the GLIBC version on your system is older than what Phoenix Code currently supports. We require at least GLIBC 2.31. Since you're using MX Linux 19.4 based on Debian 10.13, which comes with GLIBC 2.28, you might face compatibility issues.

However, you can try downloading a binary that's compatible with GLIBC 2.31 and test it on your setup. Here's the link to the specific version you can use:

[Download Phoenix Code for GLIBC 2.31](https://github.com/phcode-dev/phoenix-desktop/releases/download/prod-app-v3.8.8/phoenix-code_3.8.8_amd64_linux_bin-GLIBC-2.31.tar.gz)

Please let us know if this version works for you, or if you encounter any further issues.
